console/src/lib/AccessTokenProvider.tsx

```diff
@@ -59,14 +71,16 @@ function useAccessTokenInternal(): string | undefined {
         return;
       }
       setAccessToken(accessToken);
-    })();
+    })().finally(() => {
+      setIsRefreshing(false);
+    });
   }, [accessTokenLikelyValid]);
 
   if (error) {
     throw error;
   }
```

Comment:
If this occurs right now, a blank screen is shown with no redirection to `/login`. Do we want to just `console.error` these instead and always redirect to `/login` on failures?